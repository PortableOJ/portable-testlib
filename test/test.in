10
1 2 3 4 5 6 7 8 9 10 5
1.1
+2.2
+.3
-.4
.5


3
11111111111
11111111111 11111111111

3
10 1234567890
5 abcde
7 1234567
3
abc abc
123 123
xyz xyz
