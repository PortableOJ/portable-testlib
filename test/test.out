3
10 1234567890
5 abcde
7 1234567
3
abc abc
123 123
xyz xyz
